src/neo4j_graphrag/generation/graphrag.py

```diff
@@ -134,7 +136,10 @@ def search(
         )
         logger.debug(f"RAG: retriever_result={retriever_result}")
         logger.debug(f"RAG: prompt={prompt}")
-        answer = self.llm.invoke(prompt)
+        if chat_history is not None:
```

Comment:
I know this part isn't great. The idea I want to demonstrate is to have two different flows for chat with message history and single invokation.